### PR TITLE
Update introduction.mdx

### PR DESCRIPTION
**Pull request by akshayamadhuri**
#944  issue

### CODE REVIEW

**Comment by AvineshTripathi** [comment]
cc @jakepage91

**Review by mlabouardy — APPROVED** [review]

**Comment by AvineshTripathi** [comment]
@jakepage91 are we ready for a merge?

File: docs/getting-started/introduction.mdx [hunk 1]
```diff
@@ -42,18 +37,14 @@ Follow the links below to get the installation steps for the each cloud provider
 
 ## Contributing
 
-Komiser is written in `Golang` and is `Elv2 licensed` - contributions are always welcome whether that means providing feedback, be it through GitHub, through the `#feedback` channel on our [Discord server](https://discord.tailwarden.com) or testing existing and new features. All of the relevant information below:
-
-- [Contribute](/docs/Contributing/contribute.md)
+Komiser is written in `Golang` and is `Elv2 licensed` - contributions are always welcome whether that means providing feedback, be it through GitHub, through the `#feedback` channel on our [Discord server](https://discord.tailwarden.com) or testing existing and new features.Find all the details on how to contribute in the [Contribute](https://docs.komiser.io/community-support/contributing) section.
```
Comment (shivam-sharma7) [inline docs/getting-started/introduction.mdx:40]:
```suggestion
Komiser is written in `Golang` and is `Elv2 licensed` - contributions are always welcome whether that means providing feedback, be it through GitHub, through the `#feedback` channel on our [Discord server](https://discord.tailwarden.com) or testing existing and new features. Find all the details on how to contribute in the [Contribute](https://docs.komiser.io/community-support/contributing) section.
```

**Review by shivam-sharma7 — APPROVED** [review]

**Review by shivam-sharma7 — APPROVED** [review]

**Comment by jakepage91** [comment]
Sorry I did have a review but it was `pending` until Mohamed pointed it out to me today, but it was a small thing so yeah, let's merge.

**Pull request merged**